src/non-profits.html

```diff
@@ -78,28 +78,28 @@ <h2 class="section-heading">Our Clients</h2>
     <p>
       <div class="non-profits-wrap">
         <div class="client-testimonial">
-          <div class="col-2">
+          <div class="col-xs-6">
             <img src="img/non-profits/jaspals-voice-logo.png" alt="Jaspal's Voice Logo Image">
           </div>
-          <div class="col-2 speech-bubble-left">
+          <div class="col-xs-6speech-bubble-left">
```

Comment:
@lili2311 Thanks for the review, that is rectified now and let me know if you are happy for me to merge.